test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "mocha": true
+  }
+}
```

Comment:
Meh. Is an extra file in `test/` really worth it just for one option? Just like we did with [JSHint](https://github.com/yeoman/generator-node/blob/b25e50b89894093901116e442f9f2c1f03bb5317/.jshintrc#L10) I prefer this to go in the main `.eslintrc`.